doc/user_guide/user_guide.md

```diff
@@ -690,6 +693,9 @@ not in import statement itself.
   more complex configuration of SASL authentication. It should refer to the file
   stored inside a bucket in Exasol BucketFS.
 
+* ``SASL_KRB5CONF_LOCATION`` - It is the location of the custom ``krb5.conf`` file. 
+  It should refer to the file stored inside a bucket in Exasol BucketFS. 
```

Comment:
```suggestion
* ``SASL_KRB5CONF_LOCATION`` - It is the location of the custom ``krb5.conf`` file. It should refer to the file stored inside a bucket in Exasol BucketFS. 
```

Reply:
As (at least I) am always wondering about the paths in BucketFS, i.e. where to start and whether there must be a slash prefix, etc.: Could you give an example?